src/client/eppo-precomputed-client.spec.ts

```diff
@@ -430,7 +430,8 @@ describe('EppoPrecomputedClient E2E test', () => {
     });
 
     it('Fetches initial configuration with parameters in constructor', async () => {
-      client = new EppoPrecomputedClient(precomputedFlagStore, requestParameters);
+      client = new EppoPrecomputedClient(precomputedFlagStore);
+      client.setSubjectAndPrecomputedFlagsRequestParameters(requestParameters);
```

Comment:
Should this be an (optional) part of the constructor?

Reply:
It's subjective, I think.

I am extracting subject info from the request parameters and setting them to instance variables with this method. It feels more explicit and transparent to me when it occurs through a method called `setSubjectAndPrecomputedFlagsRequestParameters` vs passing in request parameters at instantiation and magically having the subject variables set behind the scenes.

Reply:
Yeah I think the way it's handled upstream is fine 👌